WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -13,6 +14,26 @@ namespace WalletWasabi.Hwi.Parsers;
 
 public static class HwiParser
 {
+	/// <summary>
+	/// Parse the wallet's path from the response.
+	/// </summary>
+	/// <param name="path">The wallet path which come from HWI enumerate command.</param>
+	/// <param name="model">The hardware wallet model.</param>
+	/// <returns><c>true</c> if the path matches the model's regex, <c>false</c> otherwise.</returns>
+	public static bool ValidatePathString(HardwareWalletModels model, string path)
```

Comment:
You only use this in the tests. You should move this logic out of the production code.

Reply:
Alright.